src/importer/tests/test_util_azure.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import os
+from pathlib import Path
+
+from azure.core.exceptions import ResourceExistsError
+from django.conf import settings
+from django.test import TestCase
+
+from importer.util_azure import (
+    get_container_client,
+    get_blob_container_client,
+    store_object_on_storage_account,
+)
+
+log = logging.getLogger(__name__)
+
+def create_blob_container(container_name):
+    blob_service_client = get_container_client()
+    try:
+        container_client = blob_service_client.create_container(container_name, public_access=None)
+    except ResourceExistsError:
+        pass
+    container_client = blob_service_client.get_container_client(container=container_name)
+    return container_client
+
+class APITest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        create_blob_container(settings.AZURE_CONTAINER_NAME_BAG)
+
+    def test_create_blob_temp_url(self):
```

Comment:
Je test hiermee niet of je een blob temp url kan genereren, dus deze method name zou ik aanpassen naar wat je daadwerkelijk test.